src/jvmTest/kotlin/huge.kt

```diff
@@ -1,15 +1,15 @@
 package kotlinx.html.tests
 
 import kotlinx.html.*
-import kotlinx.html.stream.*
-import org.junit.*
-import kotlin.system.*
+import kotlinx.html.stream.appendHTML
+import org.junit.Test
+import kotlin.system.measureTimeMillis
 
 open class HugeStreamingBenchmark {
     @Test
     fun testName() {
         val sb = StringBuilder(8192)
-        val count = 10000000
+        val count = 100
```

Comment:
Why so great size decrease?

Reply:
Because it takes plenty of time with no purpose. It's not a unit test just because it doesn't check anything, but it takes several minutes on TC. With `100` iterations it takes less time and using "manual-run" we still can adjust the number of iterations and check the time.